api/agent/agent_test.go

```diff
@@ -448,14 +448,6 @@ func TestSubmitError(t *testing.T) {
 	if err == nil {
 		t.Fatal("expected error but got none")
 	}
-
-	if cm.Status != "error" {
-		t.Fatal("expected status to be set to 'error' but was", cm.Status)
-	}
-
-	if cm.Error == "" {
-		t.Fatal("expected error string to be set on call")
-	}
```

Comment:
can we still test this? we can make the async lightspeed mode call.End optional?

Reply:
added a listener to check/test this now...